style.css

```diff
@@ -0,0 +1,565 @@
+
+  .project-container1 {
+    display: grid;
+    grid-template-columns: repeat(3, auto);
+    padding-left: 6.5%;
+    margin-top: -19%;
+    justify-self: center;
+    align-self: center;
+  }
+
+  .project-container2 {
+    display: grid;
+    grid-template-columns: repeat(3, auto);
+    padding-left: 7.5%;
+    margin-top: 1%;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 120px 20px;
+    gap: 12px;
+    width: 100%;
+    height: 1195px;
+    background: #1c1a19;
+    background-image: url(images/Group3.svg);
+    background-repeat: no-repeat;
+    background-position: 102% 10%;
+    position: relative;
+    top: 100px;
+  }
+
+  .about-me h1 {
+    font-family: "Crete Round", serif;
+    font-size: 72px;
+    font-weight: 400;
+    color: #ff6b00;
+    padding-left: 5%;
+    width: 40%;
+  }
+
+  .details {
+    top: 24px;
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #f4f5f7;
+    padding-left: 5%;
+    width: 40%;
+  }
+
+  .resume {
+    padding-left: 5%;
+  }
+
+  .resume .res {
+    display: flex;
+    padding: 10px;
+    border: 1px solid #ff6b00;
+    width: 150px;
+    color: #fff;
+    background-color: #ff6b00;
+    text-decoration: none;
+    font-size: 16px;
+    font-family: "Inter", sans-serif;
+  }
+
+  .about-me-hr {
+    align-self: center;
+    padding-top: 10%;
+    width: 92%;
+    padding-left: 1%;
+  }
+
+  .experience-container {
+    display: grid;
+    grid-template-columns: 70% 80% 80%;
+    align-self: center;
+    justify-self: center;
+    margin-left: 20%;
+    justify-content: space-evenly;
+    margin-top: 15%;
+  }
+
+  .experience2 {
+    margin-left: 15%;
+  }
+
+  .experience3 {
+    margin-left: 5%;
+  }
+
+  .experience1 ul,
+  .experience2 ul,
+  .experience3 ul {
+    list-style: none;
+  }
+
+  .experience1 h4,
+  .experience2 h4,
+  .experience3 h4 {
+    font-size: 32px;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-weight: 700;
+    padding-left: 10%;
+  }
+
+  .experience1 ul li,
+  .experience2 ul li,
+  .experience3 ul li {
+    display: flex;
+    align-self: center;
+    justify-self: center;
+    margin-top: 5%;
+  }
+
+  .form-container {
+    height: 700px;
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+    justify-self: center;
+    align-self: center;
+    padding-left: 1.5%;
+  }
+
+  .form-div {
+    display: flex;
+    flex-direction: column;
+    width: 150%;
+    height: 90%;
+    padding-top: 60%;
+    margin-left: -50%;
+  }
+
+  .form-container .creator h2 {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    width: 60%;
+    height: auto;
+    font-family: "Inter", sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    font-style: normal;
+    color: #262626;
+    line-height: 52px;
+    padding-top: 20%;
+    padding-left: 8%;
+  }
```

Comment:
- [ ] Your portfolio looks great with the styling you've implemented :+1:, but it's important to adhere to the project's requirements by matching your design to the provided Figma template. Specifically, please ensure that the title of your contact me section is aligned with the corresponding fields in the template, and also that the social icons are centered on your page. By doing this, you'll be able to maintain consistency with the design specifications, which is necessary for the successful completion of the project.

**Your design**

![Screenshot from 2023-05-04 14-13-37](https://user-images.githubusercontent.com/105117832/236306964-0fbae272-de56-4635-b009-6e7af48c5a3e.png)

**Figma**

![Screenshot from 2023-05-04 14-19-59](https://user-images.githubusercontent.com/105117832/236307119-2aa9adce-fec3-4ff6-9c69-c385dd38cb53.png)